beta/src/pages/blog/all.tsx

```diff
@@ -67,5 +68,11 @@ export default function Archive() {
 Archive.displayName = 'Index';
 
 Archive.appShell = function AppShell(props: {children: React.ReactNode}) {
-  return <Page routeTree={blogIndexRecentRouteTree} {...props} />;
+  return (
+    <Page
+      displaySidebar={false}
```

Comment:
instead of adding `displaySidebar` as true to all layout pages except blog would it be good if we add `hideSidebar` to `Layoutblog` alone?

Reply:
Thank you for your review, So I created a new layout for blog pages now.